website/docs/general/community.md

```diff
@@ -8,6 +8,19 @@ keywords:
 description: This article provides useful information about Apache APISIX's Community, including 2 ways to join the Slack channel.
 ---
 
+Apache APISIX is a project maintained by community members under the Apache Software Foundation.
+It is an open source project that abides by the [Apache code of conduct](https://www.apache.org/foundation/policies/conduct.html). Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API Gateway. It handles traditional north-south traffic, as well as east-west traffic between services.
+
+## Apache APISIX Community
+
+Apache APISIX is an open source infrastructure software company that provides API processing and analytics with products and solutions for microservices and real-time traffic processing; such as API gateway, k8s ingress controller, and Service Mesh.
```

Comment:
```suggestion
Apache APISIX is an open source infrastructure software project that provides API processing and analytics with products and solutions for microservices and real-time traffic processing; such as API gateway, k8s ingress controller, and Service Mesh.
```

I think the project would be better. Since Apache APISIX is a project only.